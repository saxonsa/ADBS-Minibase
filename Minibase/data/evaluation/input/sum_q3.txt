Q(x, SUM(t * x)) :- R(x, y, z), S(x, w, t), x >= 5
