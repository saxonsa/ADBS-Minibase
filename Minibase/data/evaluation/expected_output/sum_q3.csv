5, 20
8, 120
